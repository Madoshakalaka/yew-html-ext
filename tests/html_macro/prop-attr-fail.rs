fn main() {
    ::yew_html_ext::html! {
        <div
            y=69
            #[foo(bar)]
            x=420
        />
    };
}
