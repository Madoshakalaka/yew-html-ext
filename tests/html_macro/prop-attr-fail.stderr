error: only the `#[cfg]` attribute is permitted on props
 --> tests/html_macro/prop-attr-fail.rs:5:13
  |
5 |             #[foo(bar)]
  |             ^^^^^^^^^^^
